hvplot/converter.py

```diff
@@ -2637,6 +2625,33 @@ def _process_gridded_args(self, data, x, y, z):
             not_found = [dim for dim in dimensions if dim not in self.variables]
             _, data = process_derived_datetime_pandas(data, not_found, self.indexes)
 
+        print(self.tiles, self.output_projection)
+        if self.tiles and self.output_projection is not False:
```

Comment:
Should this only be done for gridded datasets? I would hope this would also be done for tabular datasets, like in https://examples.holoviz.org/gallery/gull_tracking/gull_tracking.html

Reply:
Ah you're right. Was only thinking of gridded to maps only.